openingd/dualopend.c

```diff
@@ -3923,7 +3923,7 @@ check_future_dataloss_fields(struct state *state,
 }
 
 
-static void do_reconnect_dance(struct state *state)
+static bool do_reconnect_dance(struct state *state)
 {
```

Comment:
I suspect you did this because your previous commit did `return false` and that didn't compile? :)